src/resmom/stage_func.c

```diff
@@ -1051,8 +1056,11 @@ stage_file(int dir, int	rmtflag, char *owner, struct rqfpair *pair, int conn, cp
 		DBPRT(("%s: simple copy, remote/stagein\n", __func__))
 		rc = copy_file(dir, rmtflag, owner, source,
 			pair, conn, stage_inout, prmt);
-		if (rc != 0)
+		if (rc != 0) {
+			snprintf(log_buffer, sizeof(log_buffer), "remote stagein failed for %s directory, owner=%s, src=%s ", dname, owner, source);
+			log_err(-1, __func__, log_buffer);
```

Comment:
I think we should send errno here instead of -1 in log_err. Sending -1 especially in case of windows will call GetLastError which could give some random error code as no msdn function is called just before this logging.

Reply:
Logging prmt (source path of file) will help. Do we need dname.